modules/docs/mcp/index.ts

```diff
@@ -0,0 +1,133 @@
+import {McpServer} from '@modelcontextprotocol/sdk/server/mcp.js';
+import {bestPracticesResourcesFiles, type BestPracticesResourceFile} from './config.js';
+
+// MDX content will be injected at build time
+declare const MDX_CONTENT: Record<string, string>;
```

Comment:
I struggled here a little, our build script will inline the mdx here. I think that will be pretty bulletproof. Make the dist folder bigger, but I don't know of a better way to make sure downstream users can reliably get the file contents